src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
Lets remove this class, we dont neet in in PR(You can create test classes, but don't push them to PR-s)